templates/confirmation/confirm_new_email_address_body.txt

```diff
@@ -0,0 +1,17 @@
+Hi!
+
+Our records show that you requested a change to the email address you use to sign in to Zulip.
+Please click the link below to activate your new address.
+
+{{ activate_url }}
+
+
+{% if verbose_support_offers %}
+Feel free to give us a shout at <{{ support_email }}> if you have any questions.
+{% else %}
+If you did not request this change, please contact your Zulip administrator at <{{ support_email }}>.
+{% endif %}
+
+Cheers,
+
+The Zulip Team
```

Comment:
I think it'd be good to move the `verbose_support_offers` boilerplate into a central templates/zerver/include/verbose_support.html sort of thing, since we have this in N places and I think it's usually the exact same text.  Probably makes sense to do this as a follow-up project (or maybe a project to combines things for the templates other than this one and we can convert this one depending on merge order).